linux/keyman-config/keyman_config/convertico.py

```diff
@@ -109,7 +109,7 @@ def extractico(kmxfile):
     return True
 
 
-def main(argv):
+def main():
     if len(sys.argv) != 2:
```

Comment:
From line 125 it looks like you no longer expect an argument to be passed in.

Reply:
It's not for Python.  The magic is in line 124/125, so this method can named anything with whatever parameters.